client/src/App.js

```diff
@@ -1,31 +1,53 @@
-import { useEffect, useState } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Users from './Components/Users';
+import { useState, useEffect } from 'react';
+// import './Components/form.css';
 
-function App() {
-  const [serverData, setServerData] = useState("");
 
-  useEffect(() => {
-    async function getServerData() {
-      const resp = await fetch('/api/hello');
-      const data = await resp.json();
 
-      console.log('Data from server:', data);
 
-      setServerData(data.message);
-    }
+function App() {
+  const [user, setUser] = useState('');
+  const [isAuthorized, setIsAuthorized] = useState(false);
+  const [status, setStatus] = useState(false);
```

Comment:
It's not clear to me how "user", "isAuthorized", and "status" are all used. They feel like 3 separate ways of saying the same thing?

Reply:
user is being used to hold the value of username when verified and status is being used to show/hide the login window